apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -39,7 +45,13 @@
                                 <<"channels.update_ack">>        -> update_ack;
                                 <<"channels.shutdown_sign">>     -> shutdown;
                                 <<"channels.shutdown_sign_ack">> -> shutdown_ack;
-                                <<"channels.leave">>             -> leave
+                                <<"channels.leave">>             -> leave;
+                                <<"channels.close_solo_tx">>     -> close_solo_tx;
+                                <<"channels.close_solo_sign">>   -> close_solo_tx;
+                                <<"channels.slash_tx">>          -> slash_tx;
+                                <<"channels.shash_sign">>        -> slash_tx;
```

Comment:
```suggestion
                                <<"channels.slash_sign">>        -> slash_tx;
```
?

Reply:
I admit I did not understand yet where these different binaries are hooked (I see similar binaries e.g. `<<"channels.slash_tx">>`, `<<"channels.shash_sign">>`, `<<"channels.slash">>`) and which test is meant to fail if any do not match.

Reply:
:)